class.jetpack-plan.php

```diff
@@ -46,6 +50,7 @@ class Jetpack_Plan {
 			'supports' => array(
 				'akismet',
 				'recurring-payments',
+				'premium-content/container',
```

Comment:
Do we need to cover this one here since it's not available in Jetpack nor in Core?

Reply:
I think so, as long as we want to set that this block requires a personal plan in Jetpack sites. Let's check it.

Reply:
Is the block coming to Jetpack in the near future? If so, couldn't we handle it then?

Reply:
oh right. It doesn't make sense since this block won't be able for Jetpack sites in the current state. Let me remove it.